intercom/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -4,7 +4,7 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 Anypoint Connector for Intercom (Intercom Connector) synchronizes data and automates business processes between Google Sheets and third-party applications, either on-premises or in the cloud.
 
-Google Sheets is a real-time spreadsheet tool with powerful functions to help teams efficiently perform many functions ranging from helping sales teams collaborate on leads or opportunities to HR teams collaborate on candidates for interviewing and hiring.
+Intercom is an enterprise communication tool to help customers easily learn about the company’s products and contact the service and support teams, via a chatlike interface. 
```

Comment:
Can we change this to -- "Intercom is an enterprise-grade messaging platform to help customers easily learn about the company’s products and contact the service and support teams, via a conversational chat-like interface."

Reply:
Okay!